sdk/storage/Azure.Storage.DataMovement/tests/Shared/TransferDirection.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.DataMovement.Tests
+{
+    public enum TransferDirection
```

Comment:
This was originally called `TransferType` and defined on `DataMovementTestBase`. Renamed the enum to avoid compiler ambiguity with an enum of the same name in public API on DataMovement package.